src/CLICommand/parameter.ts

```diff
@@ -0,0 +1,59 @@
+export type ParameterName = string;
+export type ParameterType = 'single-value' | 'boolean' | 'array';
+
+export interface ParameterData {
+	name: ParameterName;
+	aliases: ParameterName[];
+	description: string;
+	type?: ParameterType;
+	default?: string;
+	required?: true;
+	forbiddenConjunctionParameters?: ParameterName[];
+	requireedConjunctionParameters?: ParameterName[];
```

Comment:
spelling nit: extra E in required

Reply:
c0f61a7a2ad689c94578a24172c72cb016135646